### DISCUSSION TITLE
RQ2: coding frequency by collection

### DISCUSSION THREAD

**Discussion by nickdgardner** [ohtap/ohtap#32] [General]
@ebf77 
As requested in today's meeting, here is a more legible table comparing "rape cluster events" within each collection as a % of that collection, now including raw counts that the percentages are based on. The table is based solely on NVivo coding output, not raw keyword searches, and an interview is considered to have a "rape cluster event" if it contains an event extent that was coded as "Topic\Sexual assault or rape". 

TODO next:
•make a similar table for "Topic\Sexual harassment" and other codes of interest
•add full collection names rather than just the abbreviations
-first need to establish a canonical mapping of ABBREVIATION -> COLLECTION_NAME  (pairs like "HM":"History Makers") and put it on the Drive

[Notebook link](https://github.com/ohtap/ohtap/blob/master/Research%20Question%202/RQ2_code_frequency_by_collection_1-15-21.ipynb)  
<img width="773" alt="Screen Shot 2021-01-15 at 7 18 44 PM" src="https://user-images.githubusercontent.com/36652026/104796075-4a517600-5768-11eb-90dc-bf3bb3a68610.png">

**Comment by ebf77** [comment]
Thanks and great point about a mapping of names beyond abbreviations. I’ll circulate something and get feedback.

Meantime, I’m suspicious about some of the numbers, e.g. BWSP with only 2 rape codes. Looks like only JZ has coded them and I can’t see these yet. And we know HM as a high percent of hits, but maybe most are harassment. We’ll have to try when all coding is done. But this exercise may reinforce my sense that NVivo – at least with current coding -- may not suffice to tease out narratives/terms.

Another display would be the percent of all rape codes that appear in each collection. Looks like 287 total in my current Nvivo, so what percent of those are in HM, BWSP, etc.

When you next revise this table, I request order in descending percent,ie from high to low. 

Make sure that Oklahoma is included. Ultimately all collections in each institutions, but also the breakdown of collections.

**Comment by ebf77** [comment]
Hi Nick,

I replied on GitHub re: the table below but want to add two things:

If you remind coders to upload latest version, be sure they first enter any corrections from the response grid, as requested!

And, here is the location of the current abbreviated names of collections.  https://docs.google.com/spreadsheets/d/181genY7XiNdFs9ZXNCyP4UvKZHAYRl3gTXFtoZYw6HU/edit#gid=258071360

My suggested revisions are attached. I’ll cc: Natalie to weigh in when she has time. Not a rush. We can use current shorthands for now. Be sure to include Oklahoma.

Gratefully,
Estelle
